server/server/controllers/userController.js

```diff
@@ -89,10 +89,42 @@ const getDishItems = async (req, res) => {
   }
 };
 
+// search for Dish ok
+const searchDishes = async (req, res) => {
+  try {
+    const { minPrice, maxPrice, category, name } = req.query;
+
+    // Define the search criteria
+    const searchCriteria = {};;
```

Comment:
please remove the extra semicolon in line 98.

Reply:
Done

Reply:
Done